src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+function Calculator() {
```

Comment:
- Following the project requirements, the `Calculator.js` component should be a class-based component. Please try to refactor your component to a class-based component.